eclcli/dh/dhclient/client.py

```diff
@@ -10,7 +10,7 @@
 import socket
 import time
 
-from keystoneclient import adapter
+from keystoneauth1 import adapter
 from oslo_utils import importutils
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    from oslo_utils import importutils
except ImportError:
    from oslo.utils import importutils
```